lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -878,8 +878,9 @@ func dockerContainerCreateParamsToPortlayer(cc types.ContainerCreateConfig, laye
 	// working dir
 	config.WorkingDir = cc.Config.WorkingDir
 
-	// attach
-	config.Attach = cc.Config.AttachStdin || cc.Config.AttachStdout || cc.Config.AttachStderr
+	// attach.  Always set to true otherwise we cannot attach later.
+	// this tells portlayer container is attachable.
+	config.Attach = true
```

Comment:
Yes.  Container create doesn't prevent a container from being attachable.  We allow containers to be non-attachable but for "docker" containers, they will be attachable.